pkg/netmap/context.go

```diff
@@ -0,0 +1,74 @@
+package netmap
+
+import (
+	"errors"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
+)
+
+const defaultPivot = 0x1234
```

Comment:
Are there any cases when default pivot is being used?

Reply:
Did you mean "non-default" ?

Reply:
No, I wonder about default value. 

I assume that `ContainerID` used as a pivot to find container nodes, `ObjectID` used as a pivot to sort container nodes in order. So when default pivot is being used? Is my assumption right?

Reply:
Made pivot a parameter to `GetPlacementVectors`.
We use only single pivot right now, because there is no object-related info during generating placement policy.
I am not sure if selection of proper node for placing an object belongs here.